app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardVMStep/PlanWizardVMStep.scss

```diff
@@ -0,0 +1,3 @@
+.toolbar-pf-filter .form-control.large-filter-input {
+  width: 250px;
+}
```

Comment:
My guess is that we'll want to apply this class to all of our table filter inputs.  I know it doesn't matter, but for organization's sake, can we move this further up in the application structure?

Reply:
Actually, we can do this later if an issue is actually created to modify the other table filter inputs